### PR TITLE
oil: update to 0.8.7.

### PR DESCRIPTION
**Pull request by UsernameRandomlyGenerated**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by ericonr** [comment]
I'm reasonably sure the `vsed` line isn't doing anything, I think we should remove it.

**Comment by ericonr** [comment]
This isn't linked against system python at all.

File: srcpkgs/oil/template [hunk 1]
```diff
@@ -1,18 +1,17 @@
 # Template file for 'oil'
 pkgname=oil
-version=0.8.6
+version=0.8.7
 revision=1
 build_style=configure
-configure_args="--prefix=/usr $(vopt_with readline)"
-hostmakedepends="$(vopt_if readline readline-devel) python"
+configure_args="--prefix=/usr"
```
Comment (ericonr) [inline srcpkgs/oil/template:6]: Why did you remove readline?

Also, the commit message should note that these changes are being made.

Comment (UsernameRandomlyGenerated) [reply srcpkgs/oil/template]: My bad, I thought I did that one only on personal branch for testing.
